modules/ossm-threescale-anatomy-of-the-configuration.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-anatomy-of-the-configuration_{context}"]
+= The anatomy of the 3scale WebAssembly module configuration
```

Comment:
Maybe make this a gerund heading since it's followed by a procedure? e.g. "Configuring the 3scale WebAssembly module" ?